packages/nextjs/components/Dice.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useCallback, useRef, useState } from "react";
+
+export const Dice = ({ onRoll }: { onRoll: () => void }) => {
+  return (
+    <div className=" mx-10">
+      <div className="card-body flex">
+        <div className="h-80  bg-black opacity-70 rounded-md"></div>
+        <div className="flex w-full justify-center p-3">
+          <span className="text-xl"> Roll a 0, 1, or 2 to win the prize! </span>
+        </div>
+        <button onClick={onRoll} className={` btn btn-secondary btn-xl normal-case font-xl text-lg`}>
+          Roll the dice
+        </button>
+      </div>
```

Comment:
Add "Rigged roll" button and contract info here

Reply:
I will accomplish this task. What are your thoughts on rolling the dice using the "Default" option, the "Basic Dice" button for standard dice, and the "Rigged Dice" for a controlled outcome?

Reply:
Leave it as it was before for now, "Roll the dice" for default and "Rigged Roll" for rigged. We can change it later if needed